src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarm.java

```diff
@@ -473,6 +538,7 @@ private Upload upload(File file, Project project, AWSDeviceFarmUploadType upload
                     writeToLog(String.format("Upload %s succeeded", file.getName()));
                     break;
                 } else if ("FAILED".equalsIgnoreCase(status)) {
+                    writeToLog(String.format("(Error Message: ) %s)", describeUploadResult.getUpload().getMetadata()));
```

Comment:
Nit: I've never seen logging in this style format, with these floating ")". They're unbalanced, btw. ;)